Utilities/DataLoading/EdFi.LoadTools/BulkLoadClient/Application/BulkLoadClientConfiguration.cs

```diff
@@ -171,7 +171,9 @@ string ResolvedUrl(string url)
             {
                 return apiMode == ApiMode.YearSpecific
                     ? Regex.Replace(
-                        url, @"(?<year>\d{4})\b", configuration.GetValue<string>("OdsApi:SchoolYear"), RegexOptions.None)
+
+                        // https://regex101.com/r/lgugB1/1
```

Comment:
Can you update this to https://regex101.com/r/KywmUK/1 everywhere to reflect your changes to the Regex replace.

Reply:
done